### ISSUE TITLE
LED status flickering intermittently when LED is on in cockpit

### ISSUE THREAD

**Issue by DaddyDoobs** [DCS-Skunkworks/dcs-bios#184]
LED status from the AH-64D and F-15E have been intermittent. I have noticed this while using the "Master Caution/Warning" lights in both pits. The LED status in DCS-BIOS will not stay "TRUE" when the light is on in pit. It will come on and then right back off ever few secs. I have tried to troubleshoot the problem over the last few months expecting this to be an export issue with DCS itself but I am at a dead end.

Is there something wrong on my end? I don't have this issue with any other module. This issue has been ongoing for about 6 months. About the the time I updated DCS-BIOS for the Apache release, it has done this. Not a game breaker, but still something that bugs me.

Anything that might help me solve this problem and relay it back to the contributors of this project would be greatly appreciated.

**Comment by WarLord211** [comment]
tested with bort? what the LED in BIOS do?

**Comment by WarLord211** [comment]
from modelviewer tests the lights have no second color so they should work(not testet ingame), range is from 0.1 to 1 (model argument) Bios goes ON if <0.3. also they not dimable so not BIOS related from my side.

my first intention were: bad connection, low current, bad led or lag in code.

**Comment by DaddyDoobs** [comment]
I can get the LED number after work, but doing some testing last night, any light status exported from BIOS to the apache or strike eagle have the same effect. I show on and stable from DCS BIOS, but exporting to my device, shows intermittent.

Test again with Hornet with same device and multiple lights and found no problem.

**Comment by DaddyDoobs** [comment]
> tested with bort? what the LED in BIOS do?

I can't get the number now, but it was the output integer for the master caution on the strike eagle. Same for apache.

On a side note, I tested text export for the F15E UFC, and the flickers on and off too, just like LED status.

Hope this helps.

**Comment by charliefoxtwo** [comment]
> The LED status in DCS-BIOS will not stay "TRUE" when the light is on in pit
What tool are you using to verify this?

> This issue has been ongoing for about 6 months. About the the time I updated DCS-BIOS for the Apache release, it has done this.
How did you update dcs-bios? I'll mention I use the master caution output in the apache in my setup and I've never had any issues with it. Are you on the latest version now, or the version from 6 months ago when you last updated?

**Comment by DaddyDoobs** [comment]
> What tool are you using to verify this?

I was using the DCS-BIOS app through Google Chrome dev mode. This worked for a while but Google stopped supporting that version. Instructions to install and use are still out of date for that.

Now I am using streamdeck which is where my problem shows up. I have been trying to narrow down to determine if it's streamdeck, DCS-BIOS, or DCS export.

> How did you update dcs-bios?

I removed all the files and inserted the newest. I then added the export line for DCS-BIOS into my export.lua.

I am using the newest version of DCS-BIOS.

**Comment by DaddyDoobs** [comment]
@charliefoxtwo Is there another way to see raw exports from DCS into BIOS? Is there an updated Google Chrome app for this since it's not supported by Chrome anymore?

**Comment by charliefoxtwo** [comment]
Yup! Check out [Bort](https://github.com/charliefoxtwo/Bort) (written by yours truly).

The guy from Total Controls put together a little instructional video for setting up dcs-bios - here's a timestamped link which includes instructions for setting up Bort https://youtu.be/LsAnP6H9BMU?t=387

**Comment by DaddyDoobs** [comment]
> Yup! Check out [Bort](https://github.com/charliefoxtwo/Bort) (written by yours truly).

Awesome! Thank you. This will help me narrow the problem down.

**Comment by charliefoxtwo** [comment]
Any luck?

**Comment by DaddyDoobs** [comment]
@charliefoxtwo Hello! Sorry about the delay in my response. I downloaded your software Bort (which is great by the way! Thank you!) and I tested a few LED and button status exports. Everything worked as it should in Bort. It showed the LED status light stayed at 100% when on in Bort. So my problem lies within the export to my streamdeck. This is just odd since the F-15E and AH-64D are the only two modules that I have issues with exporting with DCS-BIOS.

Thank you for your help and I will make sure to use your software to help me troubleshoot in the future.
